submissions/innazozulia/DOM practice/main.js

```diff
@@ -0,0 +1,153 @@
+  },
+  {
+    name: 'Beedrill',
+    imgSrc: './img/Kakuna.jpeg',
+    description: 'Beedrill is extremely territorial. No one should ever approach its nest—this is for their own safety. If angered, they will attack in a furious swarm.'
+  },
+  {
+    name: 'Pidgey',
+    imgSrc: './img/Pidgey.jpeg',
+    description: 'Pidgey has an extremely sharp sense of direction. It is capable of unerringly returning home to its nest, however far it may be removed from its familiar surroundings.'
+  },
+  {
+    name: 'Rattata',
+    imgSrc: './img/Rattata.jpeg',
+    description: 'Rattata is cautious in the extreme. Even while it is asleep, it constantly listens by moving its ears around. It is not picky about where it lives—it will make its nest anywhere.'
+  },
+];
+
+const container = Document.createElement('div');
+container.classList.add('container');
+Document.body.appendChild(container);
+
+const pageName = Document.createElement('h1');
+pageName.classList.add('page-name');
+pageName.textContent = "Choose your Pokemon";
+container.appendChild(pageName);
+
+const character = Document.createElement('div');
+character.classList.add('dictionary-name');
+character.textContent = "The Character";
+container.appendChild(character);
+
+const divMain = Document.createElement('div');
+divMain.classList.add('main');
+container.appendChild(divMain);
+
+const divMenu = Document.createElement('div');
+divMenu.classList.add('menu');
+divMain.appendChild(divMenu);
+
+const navigation = Document.createElement('ul');
+navigation.classList.add('navigation');
+const nav = Document.querySelectorAll('li');
+divMenu.appendChild(navigation);
+
+pokemons.forEach((val, key) => {
+  const li = Document.createElement('li');
+  li.classList.add('list-element');
+  li.textContent = pokemons[key].name;
+  if(key == 0){
+    li.classList.add('clicked');
+    }
+    navigation.appendChild(li);
+});
+
+const divImage = Document.createElement('div');
+divImage.classList.add('image');
+divMain.appendChild(divImage);
+
+const image = Document.createElement('img');
+image.classList.add('image');
+image.setAttribute('src', pokemons[0].imgSrc);
+divImage.appendChild(image);
+
+const divDescription = Document.createElement('div');
+divDescription.classList.add('description');
+divMain.appendChild(divDescription);
+
+const descriptionMain = Document.createElement('h3');
+descriptionMain.textContent = pokemons[0].name;
+divDescription.appendChild(descriptionMain);
+
+const description = Document.createElement('p');
+description.textContent = pokemons[0].description;
+divDescription.appendChild(description);
+
+const menuList = Document.querySelector('ul');
+menuList.addEventListener('click', event => {
+  const li = Document.querySelectorAll('li');
+  li.forEach(element => {
+    element.classList.remove('cliked');
+  });
+  pokemons.forEach((val, key) => {
```

Comment:
Try to use [Array.pototype.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find?retiredLocale=uk) here in order to prevent iterating through the whole pokemons list.